src/Common/src/System/Net/Http/Http2/Hpack/HPackDecoder.cs

```diff
@@ -25,7 +25,7 @@ private enum State
 
         public const int DefaultHeaderTableSize = 4096;
         public const int DefaultStringOctetsSize = 4096;
-        public const int DefaultMaxResponseHeadersLength = HttpHandlerDefaults.DefaultMaxResponseHeadersLength * 1024;
+        public const int DefaultMaxResponseHeadersLength = 64 * 1024; // HttpHandlerDefaults.DefaultMaxResponseHeadersLength * 1024;
```

Comment:
nit: Remove comment at this point?

Reply:
I wanted to leave some reference to what the value was derived from.